As a note, unit tests are failing on the following error resolved in https://github.com/ChristopherMayes/openPMD-beamphysics/pull/142.

```
ImportError while loading conftest '/home/runner/work/lume-impact/lume-impact/impact/tests/conftest.py'.
impact/__init__.py:1: in <module>
    from .impact import Impact
impact/impact.py:75: in <module>
    STATIC_UNITS = _build_static_units()
                   ^^^^^^^^^^^^^^^^^^^^^
impact/impact.py:70: in _build_static_units
    out[k] = unit(v)
             ^^^^^^^
../../../miniconda3/envs/lume-impact-dev/lib/python3.12/site-packages/beamphysics/units.py:666: in unit
    return pmd_unit(symbol)
           ^^^^^^^^^^^^^^^^
../../../miniconda3/envs/lume-impact-dev/lib/python3.12/site-packages/beamphysics/units.py:142: in __init__
    u = self.from_symbol(unitSymbol)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../miniconda3/envs/lume-impact-dev/lib/python3.12/site-packages/beamphysics/units.py:237: in from_symbol
    raise ValueError(f"Unknown unitSymbol: {unitSymbol}")
E   ValueError: Unknown unitSymbol: Hz
```
```

--- a/docs/examples/parallel_run_example.ipynb
+++ b/docs/examples/parallel_run_example.ipynb
@@ -70,9 +70,9 @@
     "\n",
     "# Change some things\n",
     "I.header[\"Np\"] = 10000 // 2  # Make smaller problem for speed\n",
-    "I.header[\"Nx\"] = 32 // 2\n",
-    "I.header[\"Ny\"] = 32 // 2\n",
-    "I.header[\"Nz\"] = 32 // 2\n",
+    "I.header[\"Nx\"] = 32\n",
+    "I.header[\"Ny\"] = 32\n",
+    "I.header[\"Nz\"] = 32\n",
     "I.header[\"Dt\"] = 5e-13\n",
     "I.stop = 3\n",
     "\n",
@@ -342,7 +342,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "lume-impact-dev",
    "language": "python",
    "name": "python3"
   },
@@ -356,7 +356,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.8"
+   "version": "3.12.13"
   }
  },
  "nbformat": 4,

--- a/impact/impact.py
+++ b/impact/impact.py
@@ -29,7 +29,7 @@ from .interfaces.bmad import impact_from_tao
 
 
 from beamphysics import ParticleGroup
-from beamphysics.units import pmd_unit, unit
+from beamphysics.units import pmd_unit
 from beamphysics.interfaces.impact import impact_particles_to_particle_data
 
 from scipy.interpolate import interp1d
@@ -67,7 +67,7 @@ def _build_static_units():
     out = {}
     for src in (HEADER_UNITS, ELE_UNITS, RUN_INFO_UNITS):
         for k, v in src.items():
-            out[k] = unit(v)
+            out[k] = pmd_unit(v)
     out.update(EXTRA_UNITS)
     return out
 

